java/spacewalk-java.changes

```diff
@@ -1,3 +1,6 @@
+- Update tomcat jars to version 9.0.75 (for dev and test)
+- Update jetty-util to version 9.4.51 (for dev and test)
```

Comment:
Same

Reply:
I mentioned it to make clear that the effect of this is only on compiling the code locally and for CI (checkstyle and unit tests), but I can remove it of course if you prefer.